src/components/AnalysisModal.js

```diff
@@ -315,7 +315,7 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
           resetView()
           onDismiss()
         },
-        onPrevious: () => !!viewMode && resetView()
+        onPrevious: !!viewMode ? () => resetView() : undefined
```

Comment:
There is no functional difference here, correct?

Reply:
With the old version, onPrevious was never undefined, it was still a function object even if !!viewMode was false which allowed the arrow in the AnalysisModal TitleBar to show up when it shouldn't. With the second version, this is fixed as onPrevious can be undefined. This logic is consistent with other implementations of the TitleBar in the GalaxyModal as well.